tg/tdlib.py

```diff
@@ -380,12 +380,14 @@ def get_user(self, user_id: int) -> AsyncResult:
         }
         return self._send_data(data)
 
-    def get_user_full_info(self, user_id: int) -> AsyncResult:
+    def get_user_full_info(
+        self, user_id: int, block: bool = False
```

Comment:
we don't need `block` as argument here. we will always use it with `True`

Reply:
I am trying to keep it as universal API that can be used with tdlib. According to documentation, there is an `updateUserFullInfo` method which might be used to get this method async.
So actually, I'd remove block argument and make it with wait() or add `**kwargs` to each function (which will be passed to send_data) to make it consistent.

Reply:
`**kwargs ` - that is unneeded complications. `.wait()` - we don't need to bloat client's code with this. YAGNI. let's add stuff when we really need it. let's not solve imaginary problems

Reply:
I am not solving any problems, I want to keep API consistent. I have not problems with wait() and don't think it's complications or anything like that. But I agree, let's add things when we need it, I don't need block argument.